rules/msk_app_topics.go

```diff
@@ -0,0 +1,174 @@
+package rules
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	"github.com/terraform-linters/tflint-plugin-sdk/hclext"
+	"github.com/terraform-linters/tflint-plugin-sdk/logger"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/zclconf/go-cty/cty"
+)
+
+// MskAppTopics checks whether an MSK module only consumes from topics
+// defined in the module.
+type MskAppTopics struct {
+	tflint.DefaultRule
+}
+
+func (r *MskAppTopics) Name() string {
+	return "msk_consume_in_module"
```

Comment:
Let's update this name too

Reply:
> Let's update this name too

[3f383ec](https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/pull/9/commits/3f383ec4780c1a250a409dda922c566b3b480f0e)